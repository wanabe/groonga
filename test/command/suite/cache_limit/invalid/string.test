cache_limit --max LIMIT
