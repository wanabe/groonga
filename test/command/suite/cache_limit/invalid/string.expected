cache_limit --max LIMIT
[[[-22,0.0,0.0],"max value is invalid unsigned integer format: <LIMIT>"],100]
#|e| max value is invalid unsigned integer format: <LIMIT>
